dts/bindings/mipi-dbi/mipi-dbi-host.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2023 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Properties common to all MIPI DBI hosts, regardless of mode used.
+
+properties:
+  cd-invert:
+    type: boolean
+    description: |
+      Inverts polarity of command/data signal. If using 4 wire SPI mode, this
+      will cause the C/D signal to be high for commands, low for data. If using
+      3 wire SPI mode, the 9th command/data bit will be set to 1 for commands,
+      0 for data.
+
+  rst-invert:
+    type: boolean
+    description: |
+      Inverts polarity of reset signal. Signal is usually active low, this
+      property will configure it as active high
```

Comment:
Yes- for the MIPI DBI SPI driver, GPIOs would be used for this use case. However, other IP blocks that support this driver class would not use GPIOs for these signals but would still support inversion of the C/D pin or the RST pin polarity, since the polarity may vary between different displays

Reply:
understood :)